spec/rspec/runner/configuration_spec.rb

```diff
@@ -27,15 +27,16 @@
   context 'with overrides' do
     let(:overrides) do
       {
-        'map_path' => 'execution_map.yml',
+        'execution_map_path' => 'execution_maps/',
         'repo_path' => 'test',
         'predictor_class_name' => 'MyPredictor',
         'requires' => ['test.rb'],
         'diff_from' => 'HEAD~3',
         'diff_to' => 'HEAD',
         'map_expiration_period' => 1,
         'runner_class_name' => 'MyRunner',
-        'custom' => 42
+        'custom' => 42,
+        'tables_map_path' => 'tables_map.yml'
```

Comment:
no need to add this since `custom` check checks it all. There is no logic related to `tables_map_path` at all.